### ISSUE TITLE
Completions for symbol names are shown after `"` in interactive window

### ISSUE THREAD

**Issue by debonte** [microsoft/pylance-release#6040]
## Environment data

-   Language Server version: 2024.6.1
-   OS and version: Windows 11
-   Python version: 3.12.1

## Repro Steps

1. `Jupyter: Create Interactive Window`
2. In the input box, type `x = "` and note the completion suggestions

## Expected behavior

No completion suggestions, as you'd see in a `.py` file:

![image](https://github.com/microsoft/pylance-release/assets/12820898/fdea4a62-308a-436c-90a8-b1c7c14140a7)

or in a regular notebook:

![image](https://github.com/microsoft/pylance-release/assets/12820898/97c99355-717a-439c-a8c6-e3ca06ed9947)

## Actual behavior

![image](https://github.com/microsoft/pylance-release/assets/12820898/26631a36-c1ea-4a2c-ae80-5964796a60e2)

Worse yet, after typing the close quote, you get the suggestions again:

![image](https://github.com/microsoft/pylance-release/assets/12820898/5fc9f7c0-ea43-4fa1-b6a3-e7ae85c7e740)

**Comment by rchiodo** [comment]
Yeah I can repro this same thing in a notebook (if I run a cell).

This should be a jupyter extension issue I believe.

**Comment by debonte** [comment]
> That might be coming from the kernel. It does string completions. Does it repro in a notebook if you don't run any cells?

No
